packages/config/packages/config/config/devices/0x0371/zw141.json

```diff
@@ -0,0 +1,503 @@
+			"defaultValue": 150,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"35": {
+			"label": "Curtain trip time",
+			"description": "Set the run time of the curtain. Details can be found in section 4.2 of Advanced information Product Manual.",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 500,
+			"maxValue": 32767,
+			"defaultValue": 15000,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"36": {
+			"label": "Enter/exit calibration mode",
+			"description": "Set to enter/exit calibration mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Set: Exit calibration mode/Report: Not in calibration mode",
+					"value": 0
+				},
+				{
+					"label": "Set: Enter shutter mode calibration/Report: in shutter mode calibration",
+					"value": 1
+				},
+				{
+					"label": "Set: Enter Venetian mode calibration/Report: in Venetian mode calibration",
+					"value": 2
+				}
+			]
+		},
+		"37": {
+			"label": "User confirmation for calibration",
+			"description": "Set user confirmation for calibration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Default",
+					"value": 0
+				},
+				{
+					"label": "Confirm calibration",
+					"value": 1
+				}
+			]
+		},
+		"38": {
+			"label": "Calibration mode status report",
+			"description": "Get calibration mode status report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Calibration mode complete",
+					"value": 0
+				},
+				{
+					"label": "Start calibration mode, going to reference point A",
+					"value": 1
+				},
+				{
+					"label": "Reach reference point A, going to reference point B",
+					"value": 2
+				},
+				{
+					"label": "Reach reference point B, waiting for blade turn calibration",
+					"value": 3
+				},
+				{
+					"label": "Going to reference point C",
+					"value": 4
+				},
+				{
+					"label": "Calibration mode terminated",
+					"value": 5
+				}
+			]
+		},
+		"39": {
+			"label": "Curtain mode opeation",
+			"description": "Set the curtain mode operation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Shutter mode",
+					"value": 0
+				},
+				{
+					"label": "Venetian mode",
+					"value": 1
+				}
+			]
+		},
+		"40": {
+			"label": "Repositioning",
```

Comment:
Unclear. Please clarify what this does and make the label more clear. If necessary add a description.